src/plugins/intel_gpu/src/kernel_selector/kernels/stft/stft_kernel_ref.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "stft_kernel_ref.h"
+
+namespace kernel_selector {
+ParamsKey STFTKernelRef::GetSupportedKey() const {
+    ParamsKey k;
+
+    k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT64);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::BF16);
+
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::BF16);
+
+    k.EnableInputLayout(DataLayout::bfyx);
+
+    k.EnableOutputLayout(DataLayout::bfyx);
+
+    k.EnableTensorOffset();
+    k.EnableTensorPitches();
```

Comment:
Please remove this from the kernel config or update cl kernel to take into account possible paddings for all input/output buffers

Reply:
Fixed.